BrickKit/bricks/src/main/java/com/wayfair/brickkit/brick/DataModel.kt

```diff
@@ -11,15 +11,21 @@ import java.io.Serializable
  */
 abstract class DataModel : Serializable {
     @Transient
-    private val updateListeners = mutableSetOf<DataModelUpdateListener>()
+    private var updateListeners: MutableSet<DataModelUpdateListener>? = null
+        get() {
+            if (field == null) {
+                field = mutableSetOf()
+            }
+            return field
+        }
 
     /**
      * Add a [DataModelUpdateListener] to the list of listeners.
      *
      * @param updateListener the listener to add
      */
     open fun addUpdateListener(updateListener: DataModelUpdateListener) {
-        updateListeners.add(updateListener)
+        updateListeners!!.add(updateListener)
```

Comment:
could also just use ?.add but that'd hide the bug... :-)

Reply:
yeah I tried a bunch of different combos of stuff and its really just gross